xbmc/pvr/recordings/PVRRecordings.cpp

```diff
@@ -125,7 +125,7 @@ void CPVRRecordings::GetSubDirectories(const CPVRRecordingsPath &recParentPath,
       pFileItem->m_dateTime = current->RecordingTimeAsLocalTime();
 
       // Assume all folders are watched, we'll change the overlay later
-      pFileItem->SetOverlayImage(CGUIListItem::ICON_OVERLAY_WATCHED, false);
+      pFileItem->SetOverlayImage(CGUIListItem::ICON_OVERLAY_WATCHED);
```

Comment:
@Glenn-1990 ping. Why this change (and the other in this commit)? What does it fix or the other way around, what does not work without this change?